src/components/pagination-component/pagination.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import './pagination.styles.css';
+
```

Comment:
Nice thought the pagination! 
Although I have a concern while I was using it! I could only see pages that my monitor could show.. maybe check the overflow-x ? maybe change the layout a little bit? using buttons "next page" instead of all the pages?  
Food for thought!